docs/java-style-guide/readme.md

```diff
@@ -328,7 +328,7 @@ Unused or redundant imports should be removed.
 
 Static imports of project-specific methods limit readability as they
 obfuscate method provenance. Static imports are thus discouraged with
-the exception of folklore methods such as Guava `Preconditions`, and
+the exception of folklore methods such as `Collectors`, Guava `Preconditions`, and
```

Comment:
probably worth expanding this to the current permitted static imports:

https://github.com/palantir/gradle-baseline/blob/develop/.baseline/checkstyle/checkstyle.xml#L85-L91